src/js/components/awardv2/AwardV2.jsx

```diff
@@ -35,6 +35,10 @@ const awardSections = [
     {
         section: 'additional-information',
         label: 'Additional Information'
+    },
+    {
+        section: 'referenced-awards',
+        label: 'Referenced Awards'
     }
 ];
 
```

Comment:
Again, out of scope for this PR, but would love to know your thoughts about the `jumpToSection` method on this class regarding direct interaction with the DOM.